src/SDKs/Dns/Management.Dns/Properties/AssemblyInfo.cs

```diff
@@ -7,7 +7,7 @@
 [assembly: AssemblyTitle("Microsoft Azure DNS Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure DNS Resources.")]
 [assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.0.0.0")]
+[assembly: AssemblyFileVersion("2.2.0.0")]
```

Comment:
@muwaqar why go to 2.2 instead of 2.1?

Reply:
2.1 is taken by 2017-10-01 API version (changes for which should be in psSdkJson6 branch already).